src/main/java/me/creighton/encodedid/Utilities.java

```diff
@@ -1,34 +1,55 @@
 package me.creighton.encodedid;
 
+import org.jetbrains.annotations.Contract;
+
 import java.math.BigInteger;
 import java.nio.ByteBuffer;
+import java.security.SecureRandom;
 import java.util.*;
 
 import static me.creighton.encodedid.IAlphabet.LEGAL_URI_CHARACTER_SET;
 
+
 public class Utilities {
 
+  static final SecureRandom secureRandom = new SecureRandom();
```

Comment:
Minor feedback - code convention for constants is to use all-caps:

static final SecureRandom SECURE_RANDOM = new SecureRandom();

Reply:
Yep. Oversight.